RFC/src/RFC-0300_DAN.md

```diff
@@ -1 +1,179 @@
+"OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Disclaimer
+
+The purpose of this document and its content is for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community regarding the
+technological merits of the potential system outlined herein.
+
+## Goals
+
+The goal of this RFC is to describe the key features of the Tari second layer, a.k.a the Digital Assets Network (DAN)
+
+## Related RFCs
+
+* [RFC-0100: Base layer](RFC-0100_BaseLayer.md)
+* [RFC-0310: Digital assets](RFC-0310_Assets.md)
+* [RFC-0311: Second layer consensus strategies](RFC-3011_SecondLayerConsensus.md)
+* [RFC-0312: Validator nodes](RFC-3012_ValidatorNodes.md)
+
+## Description
+
+### Abstract
+
+[Digital asset]s (DAs) are managed by committees of special nodes called [Validator node]s (VNs). VNs manage digital asset state change and ensure
+that the rules of the asset contracts are enforced.
+* VNs form a peer-to-peer communication network that together defines the Tari [Digital Asset Network] (DAN)
+* VNs register themselves on the base layer and commit collateral to prevent Sybil attacks.
+* Scalability is achieved by sacrificing decentralisation. Not *all* VNs manage *every* asset. Assets are managed by
+  subsets of the DAN, called VN committees. These committees reach consensus on DA state amongst themselves.
+* VNs earn fees for their efforts.
+* Digital asset contracts are not Turing complete, but are instantiated by [Asset Issuer]s (AIs) using Digital Asset templates that are defined
+  in the DAN protocol code.
+
+### Digital Assets
+
+* Digital asset contracts are *not* Turing complete, but are selected from a set of [DigitalAssetTemplate]s that govern
+  the behaviour of each contract type. e.g. there could be a Single-Use Token template for simple ticketing systems; a
+  Coupon template for loyalty programmes and so on.
+* The template system is intended to be highly flexible and additional templates can be added to the protocol periodically.
+* Asset issuers can register TLDIs (top-level digital issuer) names on the base chain to help disambiguate similar
+  contracts and improve the signal-to-noise ratio from scam- or copy-cat contracts.
+
+An [Asset Issuer] (AI) will issue a Digital Assets by constructing a contract from on of the supported set of [DigitalAssetTemplate]s. The AI will choose
+ how large the committee of Validator Nodes will be for this DA and have the option to nominate [Trusted Node]s to be part of the VN committee for the DA.
+Any remaining spots on the committee will be filled by permissionless VNs that are selected randomly by the DAN.
```

Comment:
```suggestion
Any remaining spots on the committee will be filled by permissionless VNs that are selected according to a `CommitteeSelectionStrategy`.
```

I think that it would be beneficial to have committee selection configurable by the AI. Example strategies -- in addition to a purely random selection -- might include:
* Nodes that have <nnn ms latency
* Nodes that have at least NNN Tari available as collateral
etc

Reply:
Yes! of course. included this suggestion and elaborated on the strategy.